Add blog post about model-transparency repo (#59)
sigstore/sigstore-blog: content/new-repository-model-transparency.md

+++
title = "New repository: model-transparency"
date = "2024-03-20"
tags = ["sigstore","artificial intelligence"]
draft = false
author = "mihaimaruseac"
type = "post"
+++

> In order to strengthen the supply chain of ML, we have adopted a new

Reviewer: ```suggestion
In order to strengthen the supply chain of ML, the Sigstore TSC voted to adopt a new
```
